app/models/setting/provisioning.rb

```diff
@@ -46,6 +46,7 @@ def self.default_settings
       self.set('access_unattended_without_build', N_("Allow access to unattended URLs without build mode being used"), false, N_('Access unattended without build')),
       self.set('manage_puppetca', N_("Foreman will automate certificate signing upon provision of new host"), true, N_('Manage PuppetCA')),
       self.set('ignore_puppet_facts_for_provisioning', N_("Stop updating IP address and MAC values from Puppet facts (affects all interfaces)"), false, N_('Ignore Puppet facts for provisioning')),
+      self.set('unattended_orchestration', N_("Enable host orchestration during unattended operations"), false, N_('Enable unattended orchestration')),
```

Comment:
This is not good description, this flag for example will break discovery plugin.

Reply:
Interesting, I definitely don't want to break discovery... The motivation for this is that I just want my hosts (which are all provisioned from discovered hosts) to keep DNS in sync with the host, which is updated by the puppet fact uploads. I knew there were potential performance issues but I wasn't aware that it would break any existing behaviors. Is there a better way I can go about this so that it won't affect the discovery plugin?

Reply:
Ok I take it back, it does not break discovery. Go head but I think the name and description is a little bit confusing, this has something to do with fact importing.

Reply:
do you think something like `enable_orchestration_on_fact_import` would be more fitting?

Reply:
With a big old performance disclaimer in the description